verilog/tools/ls/README.md

```diff
@@ -89,14 +89,69 @@ to get it started up on our Verilog/SystemVerilog projects.
 
 ### Neovim
 
-Make sure to have version `0.5.0` or newer and install the [nvim-lpsconfig](https://github.com/neovim/nvim-lspconfig/) plugin.
-Enable the verible config:
+Make sure to have version `0.5.0` or newer and install the [nvim-lspconfig](https://github.com/neovim/nvim-lspconfig/) plugin.
+You can install it with the popular [vim-plug](https://github.com/junegunn/vim-plug) plugin manager by adding the following code to your ~/.config/nvim/init.vim file:
+```vim
+call plug#begin()
+Plug 'neovim/nvim-lspconfig'
+call plug#end()
+```
+Then to install it open neovim and type: ``` :PlugInstall ```
+
+After installing nvim-lspconfig, enable the verible config by appending the following script to your ~/.config/nvim/init.nvim file:
 
 ```lua
--- init.lua
-require'lspconfig'.verible.setup {}
-```
+lua << EOF
```

Comment:
Can this be trimmed down ?

To help people understand the configuration I think it is important we want to drill down as much as possible to the minimum configuration that is needed for someone to get the LSP going. Otherwise someone who is new to neovim will have a hard time to distinguish what is important ( `require'lspconfig'.verible.setup ...`) and what is noise (such as the keybindings). I suspect that the LSP configuration comes with its own stater set of keybindings anyway ? Or do we have to give the keybindings otherwise the features would not be accessible ?

We then can point them to some URL for fine-tuning keymaps etc. then it is easier to read for users here, as they only get the relevant gist. Also with the external configuration, we don't have to keep track of the neovim features that might improve and change over time.